server/controllers/list.js

```diff
@@ -1210,47 +1210,51 @@ const getArchivedItems = (req, resp) => {
       }
 
       const { items } = list;
-      const archivedItems = items.filter(item => item.isArchived);
+      const activeItems = items.filter(item => !item.isDeleted);
+      const archivedItems = activeItems.filter(item => item.isArchived);
```

Comment:
why not just one filter:
`const archivedItems = items.filter(item => item.isArchived && !item.isDeleted)`
?

Reply:
There was one, Przemo said "Split filters"  😄
https://github.com/sandstreamdev/eoc/pull/265#discussion_r310995105